### ISSUE TITLE
Missing details with V4.0 

### ISSUE THREAD

**Issue by bachtron** [andrewleech/ha_magiqtouch#24]
In the previous version, it showed what state it was in, whether it was currently running the burner and heating, or idle.
The new version only has heating and off.

Running V4.0
HASS Core: 2024.3.1

**Comment by andrewleech** [comment]
Yeah after the complete re-write by seeley of their entire cloud it's taken a huge rework to get anything working again.
v4.0 is only an initial / interim release with basic functionality, further work is ongoing as discussed in https://github.com/andrewleech/ha_magiqtouch/issues/19

**Comment by andrewleech** [comment]
The missing idle/heating/cooling states should be working now. v5.2 includes a fix from @matthew-muscat that checks the unit's runningState to show IDLE when the thermostat has reached target temperature rather than always showing HEATING/COOLING. Let me know if it's still not right on the latest version.
